source/NVDAObjects/window/winword.py

```diff
@@ -14,10 +14,6 @@
 from comtypes import COMError, GUID, BSTR
 import comtypes.client
 import comtypes.automation
-import uuid
-import operator

```

Comment:
While the imports you've removed aren't used in this file `NVDAObjects.IAccessible.winword` star imports everything from here and removing them causes failures when trying to navigate in tables. I've created #13805 to fix this - please merge it to master before this PR.